build_protos.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -e
+
+REPO_ROOT=$(git rev-parse --show-toplevel)
+
+echo Building Rust bindings. Output will be in ./steward_proto_rust/prost
+cargo run --package steward_proto_rust_build --release
+
+GO_OUT=$REPO_ROOT/steward_proto_go
+PROTO_PATH=$REPO_ROOT/proto
+
+echo
+echo Building Go bindings. Output will be in ./steward_proto_go/steward_proto
+protoc --proto_path=$PROTO_PATH \
```

Comment:
Our other repos use (a very old version of) https://github.com/bufbuild/buf for handling go proto builds. Not a requirement at the moment but something worth looking into.